package.json

```diff
@@ -3,6 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "4.3.1",
```

Comment:
follow this tutorial for setting up bootstrap properly for react https://react-bootstrap.github.io/getting-started/introduction/

Reply:
Integrated react-bootstrap & imported bootstrap in App.js + CDN stylesheet in index.html